### ISSUE TITLE
Ошибка в имени миграции

### ISSUE THREAD

**Issue by agarb7** [agarb7/my.briop#151]
Вот эта [миграция](https://github.com/agarb7/my.briop/blob/a664996255d2621f0e0c363641c5945667f398f9/migrations/m081115_132312_svedeniya_o_sebe.php).
Может привести к глюкам при применении миграций.

**Comment by tsyrya** [comment]
А какие глюки?

**Comment by tsyrya** [comment]
@agarb7

**Comment by agarb7** [comment]
Имя определяет последовательность выполнения миграций. Исходя из имени твоя миграция должна выполниться самая первая, и не сможет этого сделать т.к. нужны изменения предыдущих. 
Ты её на сервере применял?

**Comment by tsyrya** [comment]
Да применял

**Comment by agarb7** [comment]
ну теперь её уже не исправишь(

**Comment by tsyrya** [comment]
так он же кажется сам все по датам сортирует

**Comment by agarb7** [comment]
да, а даты берёт из имени, а миграция носит имя m081115_132312_svedeniya_o_sebe, что соответствует 15 ноября 2008 года.

**Comment by tsyrya** [comment]
аа, я чет затупил, извиняюсь, ну есть вариант: исправить название и исправить в базе

**Comment by agarb7** [comment]
я думаю, когда будем реп переливать, тогда почистить папку с миграциями, и таблицу миграций в БД: будем коммиты аккуратно вести, ветвлениями пользоваться, пул реквесты, ишью к комитам цеплять и т.п.. С чистого листа без варварства)
